CONTRIBUTING.md

```diff
@@ -38,13 +49,21 @@ playground rules and follow them during all your contributions.
  - For more on testing see the 
  [Coding in PANOPTES](https://github.com/panoptes/POCS/wiki/Coding-in-PANOPTES) page.
 
-
 # Code Formatting
 
 - All Python should use [PEP 8 Standards](https://www.python.org/dev/peps/pep-0008/)
    - Line length is set at 100 characters instead of 80.
    - It is recommended to have your editor auto-format code whenever you save a file 
       rather than attempt to go back and change an entire file all at once.
+   - You can also use
+     [yapf (Yet Another Python Formatter)](https://github.com/google/yapf)
+     for which POCS includes a style file (.style.yapf). For example:
+     ```bash
```

Comment:
Should we make a script for this?

Reply:
Well, maybe. You can submit an issue if you like.

Reply:
We'll leave for now.